cypress/fixtures/example.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Looks like this got reintroduced (I seem to recall you had removed it? Or was that a comment on my old PR? I agree it's not needed)

Reply:
It's auto-generated by Cypress :D

Reply:
Yikes haha. we could gitignore but if it's gonna get auto generated by cypress yah not worth the effort